oblib/taxonomy.py

```diff
@@ -15,7 +15,7 @@
 """Handles Orange button taxonomy."""
 
 import enum
-from oblib import taxonomy_types, taxonomy_misc
+from oblib.ob import OBNotFoundError
```

Comment:
To be consistent, this should be - `from oblib import ob`
```suggestion
from oblib import ob
```